controller/src/main/java/com/softserve/borda/controllers/BoardController.java

```diff
@@ -269,14 +269,14 @@ public Board getBoard(@PathVariable Long id) {
                                                                      @PathVariable Long ticketId,
                                                                      @PathVariable String boardId) {
         try {
-            BoardList oldBoardList = boardListService.getBoardListById(oldBoardListId);
-            BoardList newBoardList = boardListService.getBoardListById(newBoardListId);
+            BoardColumn oldBoardColumn = boardColumnService.getBoardListById(oldBoardListId);
```

Comment:
I think `getBoardListById` method is no longer applicable.

Reply:
Yes, you are right. Will fix